.github/workflows/beam_PostCommit_Java_ValidatesDistrolessContainer_Dataflow.yml

```diff
@@ -0,0 +1,108 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: PostCommit Java ValidatesDistrolessContainer Dataflow
+
+on:
+  schedule:
+    - cron: '30 6/8 * * *'
+  pull_request_target:
+    paths:
+      - 'release/trigger_all_tests.json'
+      - '.github/trigger_files/beam_PostCommit_Java_ValidatesRunner_Dataflow_V2.json'
```

Comment:
do we need '.github/trigger_files/beam_PostCommit_Java_ValidatesRunner_Dataflow_V2.json' ? I understand the trigger_file has 1:1 correspondence to workflow

Reply:
Thank you @Abacn , I was thinking that since Distroless depends on the same as beam_PostCommit_Java_ValidatesRunner_Dataflow_V2, it wouldn't hurt to trigger a Distroless check again when someone has a reason to trigger beam_PostCommit_Java_ValidatesRunner_Dataflow_V2. It's only running examples.